### DISCUSSION TITLE
Frigate taking up a lot of CPU, but it's probably my config

### DISCUSSION THREAD

**Discussion by gatorreina** [blakeblackshear/frigate#3417] [General]
I have seven cameras and one Coral ai. My frigate.process (s) jump as high as 75% individually and when someone is front of a camera my uswgi can go to 100%. I am using an almost identical config for all of my cameras as below. Is there something I am doing wrong that could be driving my high CPU usage?

 mqtt:
  host: minisforum1
  port: 1883
  topic_prefix: frigate
cameras:
  cochera1:
    ffmpeg:
      hwaccel_args:
        - -hwaccel
        - vaapi
        - -hwaccel_device
        - /dev/dri/renderD128
        - -hwaccel_output_format
        - yuv420p
      output_args:
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
      inputs:
        - path: rtsp://admin:pass@192.168.50.11:554/cam/realmonitor?channel=1&subtype=1 # Amcr IP5M-T1179EW subs
          roles:
            - detect
            - rtmp
        - path: rtsp://admin:pass@192.168.50.11:554/cam/realmonitor?channel=1&subtype=0 # Amcr IP5M-T1179EW subs
          roles:
            - record
    detect:
      width: 704
      height: 480
      fps: 5
    objects:
      track:
        - person
    snapshots:
      enabled: True
      retain:
        default: 4
    record:
      enabled: True
      events:
        retain:
          default: 5
        pre_capture: 3
    zones:
      zone_0:
        coordinates: 0,118,42,122,53,234,99,260,276,203,383,190,673,311,896,464,896,672,0,672 #0,0,896,0,896,672,0,672 
    motion:
      mask:
        - 704,0,704,50,511,47,509,0

I am also using a coral AI USB accelerator:

detectors:
  coral:
    type: edgetpu

**Comment by NickM-27** [comment]
What CPU are you using and have you verified hwaccel is working? (Wouldn't explain the CPU jump, but could explain generally high usage)

What inference speed are you seeing in the frigate debug screen?

**Reply by gatorreina** [reply]
Beelink 8th Generation Intel i5-8279U Processor（up to 4.1GHz）,Beelink SEi8  with 8G DDR4 RAM/256GB M.2 NVME SSD,Supports 6MB Smart Cache,4K Dual HDMI,WIFI5,BT5.0

How do I verify if hwaccel is working?

Inference speed: 11.93

**Reply by NickM-27** [reply]
`intel_gpu_top` in the host or container should work
